src/Storage/Models/BackChannelAuthenticationRequest.cs

```diff
@@ -87,7 +87,12 @@ public class BackChannelAuthenticationRequest
     public string? SessionId { get; set; }
 
     /// <summary>
-    /// Gets the description the user assigned to the client being authorized.
+    /// Gets or sets the description the user assigned to the client being authorized.
     /// </summary>
     public string? Description { get; set; }
+
+    /// <summary>
+    /// Gets or sets a dictionary of custom properties associated with this instance.
+    /// </summary>
+    public Dictionary<string, object> Context { get; set; } = new();
```

Comment:
`Context` isn't a good name for this. `Properties` would maybe be better. This is because it would be an analog to the `Properties` dictionary in `AuthorizationCode`.

Also, I would be careful using a dictionary with `object` values in a storage model. Maybe using `string` values would be enough for this and all other `Properties` dictionaries.

Reply:
We use a dictionary<string, object> for the [JsonExtensionData], because it is required there. In the context of the storage model, we could use strings instead, but then we'd have to convert to and from strings as we pass the properties around, and if folks want to put complex objects into the properties, that adds some complexity. 

In the end, the dictionaries get serialized to json when they're stored. If you try to pass something that doesn't serialize, I would expect an error from the serializer, so I think the object is fine.

Reply:
Fair enough